### PR TITLE
fix: Fix build error during development using docker-compose

### PR DESCRIPTION
**Pull request by abehnamfard**
# Fix ssl error in final build stage on development docker compose

## Description

Please include a summary of the change and issue(s) fixed. Also, mention relevant motivation, context, and any dependencies that this change requires.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -93,6 +93,9 @@ ARG RAILS_ENV=production
 ENV RAILS_ENV ${RAILS_ENV}
 ENV BUNDLE_PATH="/gems"
 
+ARG NODE_OPTIONS="--openssl-legacy-provider"
+ENV NODE_OPTIONS ${NODE_OPTIONS}
+
```
Comment (pranavrajs) [inline docker/Dockerfile:98]: @abehnamfard Can you please share what the error was and how does this fix it? The same line is available in line 16-17. As far as I see there is no JS bundling action taken post this line.

@vishnu-narayanan Please take a look.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
@abehnamfard Thanks for the contribution. I've a comment to clarify the changes. Please take a look.

File: docker/Dockerfile (hunk 1 above)
Comment (abehnamfard) [reply docker/Dockerfile:98]: Lines 16 & 17 belong to the pre-build stage of the docker.
This change will fix the error on the webpack container when it is running (Build Stage).

Comment (abehnamfard) [reply docker/Dockerfile:98]: Screenshot 2024-01-19 at 10 32 26 PM

**Comment by abehnamfard** [comment]
@vishnu-narayanan Yes you're right.
I will fix it.

**Comment by vishnu-narayanan** [comment]
@abehnamfard Thank you. If you can update the PR, we can get this merged.

**Comment by abehnamfard** [comment]
Sorry for the delay.
Done.

**Review by pranavrajs — APPROVED** [review]

**Comment by vishnu-narayanan** [comment]
@abehnamfard Thank you for the changes.

**Review by vishnu-narayanan — APPROVED** [review]

**Pull request merged**